### PR TITLE
Add CapDrop generator

### PR DESCRIPTION
**Pull request by mconcas**

### CODE REVIEW

File: plancton/__init__.py [hunk 1]
```diff
@@ -263,7 +263,8 @@ def _create_container(self):
                            "Devices"     : [ dict(zip([ "PathOnHost", "PathInContainer",
                                                         "CgroupPermissions" ], x.split(":", 2)))
                                              for x in self.conf["devices"] ],
-                           "CapAdd"      : self.conf["capabilities"]
+                           "CapAdd"      : [ x.lstrip("+") for x in self.conf["capabilities"] if x and x[0]=="+" ]
```
Comment (dberzano) [inline plancton/__init__.py]: This condition is not backwards-compatible as it does not catch the case of missing `+`.

    [ x.lstrip("+") for x in self.conf["capabilities"] if x and x[0]!="-" ]

File: plancton/__init__.py [hunk 2]
```diff
@@ -263,7 +263,8 @@ def _create_container(self):
...
+                           "CapDrop"     : [ x[1:] for x in self.conf["capabilities"] if x and x[0]=="-" ]
```
Comment (dberzano) [inline plancton/__init__.py]: I would say for consistency to use what you've actually did before (my bad, I keep changing my mind):

    [ x.lstrip("-") for x in self.conf["capabilities"] if x and x[0]=="-" ]

**Review by dberzano — CHANGES_REQUESTED** [review]

File: plancton/__init__.py (hunk 2 above)
Comment (mconcas) [reply plancton/__init__.py]: You are right.

**Review by dberzano — APPROVED** [review]

**Comment by dberzano** [comment]
squash-n-merge when test is green

**Pull request merged**